store.go

```diff
@@ -6,6 +6,22 @@ import (
 	"strings"
 )
 
+var commonDocRoots = []string{
```

Comment:
This seems quite application specific. Another (future) consumer of gocommerce probably doesn't care about the majority of these.

Maybe `DiscoverStores` should take a `commonDocRoots ...string` (or slice, whatever floats your boat), and move this list to ecomscan side?

I'm not sure.. :)

Reply:
Hmmm, that might make testing more difficult, now that I look at it.

Reply:
Thanks! I am moving this function from casetool to gocommerce, because I want to use it in ecomscan as well. Let me know if you know a better place to share this common knowledge?